homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -299,7 +321,6 @@
     ATTR_TARGET_HUMIDITY: "target_humidity",
     ATTR_LED_BRIGHTNESS: "led_brightness",
     ATTR_USE_TIME: "use_time",
-    ATTR_HARDWARE_VERSION: "hardware_version",
```

Comment:
Removing hardware version from the entity state attributes is a breaking change. If that's ok, we need to add a breaking change paragraph in the PR description that briefly describes the breaking change and what users should do to cope with it.

Reply:
done